solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+struct LoadingView: View {
+    var message: String?
+    
+    var body: some View {
+        VStack {
+            ProgressView()
+            if let message = message {
+                Text(message)
+                    .padding()
+            }
+        }
+    }
+}
+
+#if DEBUG
+struct LoadingView_Previews: PreviewProvider {
+    static var textExample: String? = "Loading..."
+    static var previews: some View {
+        LoadingView(message: textExample)
+    }
```

Comment:
**[SUGGESTION]**
Check how it behaves in Dark mode...
```suggestion
    static let textExample: String = "Loading..."
    static var previews: some View {
        LoadingView(message: textExample)
            .preferredColorScheme(.light)
        LoadingView(message: textExample)
            .preferredColorScheme(.dark)
    }
```

Reply:
Yea, that's clever